spec/rmagick/image/capture_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.describe Magick::Image, "#capture" do
+  before do
+    @img = Magick::Image.new(20, 20)
+  end
+
+  it "works" do
+    # expect { Magick::Image.capture }.not_to raise_error
```

Comment:
Why are these commented out? If they already were can we remove it?

Reply:
Hmm, not sure why they were commented out, but it was in the original file. I guess I can remove them as they don't appear to be that important.

Reply:
Alright, updated.